homeassistant/components/voip/voip.py

```diff
@@ -102,8 +105,19 @@ def __init__(self, hass: HomeAssistant, devices: VoIPDevices) -> None:
         )
         self.hass = hass
         self.devices = devices
+        self.local_endpoint = local_endpoint
+        self.sip_host = local_endpoint.host
+        self.sip_port = local_endpoint.port
         self._closed_event = asyncio.Event()
 
+    def on_call(self, call_info: CallInfo):
+        """Set up state when starting a call."""
+        device = self.devices.async_get_or_create(call_info)
+        if device is not None:
+            device.current_call = call_info
```

Comment:
Is this something that we want to pass to `set_active` instead? Having a call info then indicates it's active.